### DISCUSSION TITLE
publish to Notion

### DISCUSSION THREAD

**Discussion by johnros** [quarto-dev/quarto-cli#7488] [Feature Requests]
### Description

I am currently enjoying the `quarto publish confluence` feature very much. 
I wonder, it there a way, existing or planned, to publish to Notion in a similar fashion?

**Comment by mcanouil** [comment]
As the number of services/hosts is quite big, and because `quarto publish` command is mostly a wrapper to "usual" commands, it's unlikely other services will be added due to maintenance cost of the code and because of priorities on Quarto internal bugs/features.

I strongly suggest you get acquainted with post-render script to handle the publication to any services such as Notion using its API documentation.
See:
- https://quarto.org/docs/publishing/other.html
- https://quarto.org/docs/projects/scripts.html#pre-and-post-render
